atlasdb-tests-shared/src/test/java/com/palantir/atlasdb/keyvalue/impl/RangeRequestTest.java

```diff
@@ -29,15 +29,15 @@ public void testPrefix() {
         assertThat(end.length).isEqualTo(0);
         end = RangeRequest.builder().prefixRange(new byte[] {-2}).build().getEndExclusive();
         assertThat(end.length).isEqualTo(1);
-        assertThat(end[0]).isEqualTo(-1);
+        assertThat(end[0]).isEqualTo((byte) -1);
```

Comment:
This is pretty sad, but there is no nice way around it. This test is also super bad anyway so fine with leaving as is